helm-charts/chartpress.yaml

```diff
@@ -22,11 +22,12 @@ charts:
           REQUIREMENTS_FILE: dynamic-image-building-requirements.txt
         contextPath: images/hub
         dockerfilePath: images/hub/Dockerfile
-  - name: support
-    images:
-      gcp-filestore-backups:
-        imageName: quay.io/2i2c/gcp-filestore-backups
-        valuesPath: gcpFilestoreBackups.image
```

Comment:
Unrelated to this PR, but had to be cleaned up to get chartpress to build correctly so was done as part of this PR for simplicity - this image is now built in an external repo.